lib/fastlane/plugin/revenuecat_internal/helper/versioning_helper.rb

```diff
@@ -173,6 +194,8 @@ def self.detect_bump_type(version_name_a, version_name_b)
           :new_features
         elsif change_types.include?("fix")
           :fixes
+        elsif change_types.include?("dependencies")
```

Comment:
I think we need to be careful with how we use the `dependencies` label after this... How about renaming it to `phc_dependencies` or something like that to make it more explicit? Otherwise I foresee us using this tag for some unrelated dependency.

Reply:
changed it with a new label